spec/bowlingSpec.js

```diff
@@ -0,0 +1,55 @@
+describe('Bowling', function() {
+  var bowling;
+
+  beforeEach(function() {
+    bowling = new Bowling();
+  });
+
+  it('starts with 10 pins standing', function() {
+    expect(bowling.pins).toEqual(10);
+  });
+
+  it('pins can be knocked down', function() {
+    bowling.pinsHit(2);
+    expect(bowling.pins).toEqual(8);
+  });
+
+  it('can roll a strike', function() {
+    bowling.pinsHit(10);
+    expect(bowling.strikes).toEqual(1);
+  });
+
+  it('can roll a spare', function() {
+    bowling.pinsHit(5);
+    bowling.pinsHit(5);
+    expect(bowling.spares).toEqual(1);
+  });
+
+  it('can have two rolls in one frame', function() {
+    expect(bowling.rollNumber).toEqual(1);
+    bowling.pinsHit(9);
+    expect(bowling.rollNumber).toEqual(2);
+  });
+
+  it('can move to the next frame after a second roll in a frame', function() {
+    expect(bowling.rollNumber).toEqual(1);
+    bowling.pinsHit(9);
+    expect(bowling.rollNumber).toEqual(2);
+    bowling.pinsHit(1);
+    expect(bowling.frameNumber).toEqual(2);
+  });
+
+  it('can move to the next frame after a strike', function() {
+    expect(bowling.frameNumber).toEqual(1);
+    expect(bowling.rollNumber).toEqual(1);
+    bowling.pinsHit(10);
+    expect(bowling.frameNumber).toEqual(2);
+  });
+
+  it('can have a total score after each frame', function() {
+    bowling.pinsHit(2);
+    bowling.pinsHit(3);
+    expect(bowling.currentScore).toEqual(5);
+  });
+
+});
```

Comment:
This tests are a good start, until you start having multiple expectations. If you are going to test the basic state of your object when it is created, then you don't need to test that again with every following test.

Also, try to move towards a more concise style for your test descriptions: it "moves to the next frame", it "has a total score after each frame".